core/src/main/scala/org/allenai/common/FileUtils.scala

```diff
@@ -0,0 +1,66 @@
+package org.allenai.common
+
+import au.com.bytecode.opencsv.CSVReader
+
+import java.io._
+
+import scala.collection.JavaConverters._
+import scala.io.{ BufferedSource, Codec, Source }
+
+import java.nio.charset.Charset
+
+/** Various convenient utilities for reading files and resources. */
+object FileUtils extends Logging {
+
+  /** Get a file as a non-lazy sequence of lines. */
+  def getFileAsLines(file: File)(codec: Codec): Seq[String] = {
+    logger.debug(s"Loading file ${file.getName}")
+    // use toVector to force stream to be processed
+    Resource.using(Source.fromFile(file)(codec))(_.getLines().toVector)
+  }
+
+  /** Read a CSV file as a non-lazy sequence (rows) of sequence (columns) of strings. */
+  def getCSVContentFromFile(file: File): Seq[Seq[String]] = {
+    logger.debug(s"Loading CSV file ${file.getName}")
+    val csvReader = new CSVReader(new FileReader(file))
```

Comment:
Changed to `new CSVReader(new InputStreamReader(new FileInputStream(file), codec.charSet))` as `FileReader` doesn't like to take a `CharSet`.